functions/services/getMailchimpTags.js

```diff
@@ -0,0 +1,11 @@
+function getMailchimpTags ({
+  tracking: { utm_campaign: source }
+}) {
```

Comment:
This wouldn't have worked, because you're passing it the `source` directly in the `processCalendlyEventInvitee` Cloud Function but this was still receiving the Calendly invitee.
```suggestion
function getMailchimpTags (source) {
```